bin/bwa_mem.pl

```diff
@@ -52,15 +52,15 @@
   my $options = setup();
 
  	my $threads = PCAP::Threaded->new($options->{'threads'});
-  &PCAP::Threaded::disable_out_err if(exists $options->{'index'});
+  #&PCAP::Threaded::disable_out_err if(exists $options->{'index'});
```

Comment:
Just checking this supposed to be commented.  If so, should it be removed?

Reply:
good spot, fixing, should be deleted